src/execCmd.ts

```diff
@@ -176,6 +180,10 @@ const execCmdAsync = async (cmd: string, options: ExecCmdOptions): Promise<ExecC
   return resultPromise;
 };
 
+// Create a Duration instance from process.hrtime
```

Comment:
Do you have a usecase in mind?  Just seems pretty specific to how it's used within this file.  It would need a public JSDoc comment if so, but personally I don't like exporting code that most people won't use.